Description: add saucy and trusty version number as a known release
Author: Paolo Sammicheli <xdatap1@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1300400
Forwarded: not-needed
---
Index: live-build-3.0~a57/functions/releases.sh
===================================================================
--- live-build-3.0~a57.orig/functions/releases.sh
+++ live-build-3.0~a57/functions/releases.sh
@@ -27,3 +27,5 @@ RELEASE_oneiric="11.10"		# 2013-04-01
 RELEASE_precise="12.04"		# LTS: 2017-04-01
 RELEASE_quantal="12.10"		# 2014-04-01
 RELEASE_raring="13.04"		# 2014-01-01
+RELEASE_saucy="13.10"		# 2014-07-01 
+RELEASE_trusty="14.04"		# LTS: 2019-04-01
